editor/src/components/navigator/left-pane/github-pane/github-spinner.tsx

```diff
@@ -0,0 +1,45 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+/** @jsxFrag React.Fragment */
+import { css, jsx, keyframes } from '@emotion/react'
+import React from 'react'
+import { FlexColumn } from '../../../../uuiui'
+
+export const GithubSpinner: React.FC = () => {
+  const anim = keyframes`
+        from {
+          transform: rotate(0deg);
+        }
+        to {
+          transform: rotate(360deg);
+        }
+      `
+
+  return (
+    <FlexColumn>
```

Comment:
Why do you need to put this single svg into a FlexColumn?

Reply:
This element is used in multiple files and that's how it was implemented, so I didn't want to disturb it